run_infrastructure.py

```diff
@@ -233,6 +236,9 @@ def _run_tests(
             for env_var in EXTERNAL_DATASTORE_CONFIG[datastore]:
                 environment_variables += f"-e {env_var} "
 
+    if analytics_opt_out:
+        environment_variables += f"-e ANALYTICS_OPT_OUT"
```

Comment:
No need for an f-string here.
```suggestion
        environment_variables += "-e ANALYTICS_OPT_OUT"
```